### PR TITLE
topsection

### PR DESCRIPTION
**Pull request by Shy-006**
This pr introduces the top section of the clockify website
Screenshot from 2024-12-18 12-11-31

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -17,29 +17,29 @@
...
 
-.logo img {
+.navbar__left-section--logo img {
     height: 35px;
 }
-.separator {
+.navbar__left-section--separator {
     width: 1px;
     height: 30px;
     background-color: #ddd;
```
Comment (B-TAMILSELVAN) [inline index.css]: save colors in css variables and use?
Apply at all places

**Comment by B-TAMILSELVAN** [comment]
match the UI properly

**Review by B-TAMILSELVAN — CHANGES_REQUESTED** [review]

**Comment by Shy-006** [comment]
> match the UI properly

ui matched

File: index.css (hunk 1 above)
Comment (Shy-006) [reply index.css]: clors have been stored in variables

**Comment by Shy-006** [comment]
> match the UI properly

matched properly

File: index.css [hunk 2]
```diff
@@ -1,3 +1,12 @@
+:root{
+    --Grey:rgba(144, 164, 174, 1); 
+    --White:#ddd;
+    --lightblack:#333;
+    --lightblue:#00a8ff;
+--lightgreen:#4caf50;
```
Comment (B-TAMILSELVAN) [inline index.css]: format them properly

File: index.css [hunk 3]
```diff
@@ -1,3 +1,12 @@
+:root{
+    --Grey:rgba(144, 164, 174, 1);
```
Comment (B-TAMILSELVAN) [inline index.css]: follow naming convention properly when naming colors

**Comment by B-TAMILSELVAN** [comment]
format the html and css file properly

**Comment by B-TAMILSELVAN** [comment]
resolve the merge conflicts

**Review by B-TAMILSELVAN — CHANGES_REQUESTED** [review]

File: index.css (hunk 3 above)
Comment (Shy-006) [reply index.css]: naming convention have beeon followed

File: index.css (hunk 2 above)
Comment (Shy-006) [reply index.css]: formatted

**Review by B-TAMILSELVAN — APPROVED** [review]

**Pull request merged**